p2p/providers/peerstore_provider/rpc/provider.go

```diff
@@ -11,46 +11,53 @@ import (
 	"github.com/pokt-network/pocket/p2p/providers/peerstore_provider"
 	typesP2P "github.com/pokt-network/pocket/p2p/types"
```

Comment:
NOT A BLOCKER: created #884

![Screenshot 2023-07-05 at 2 50 44 PM](https://github.com/pokt-network/pocket/assets/1892194/38dd190d-db3a-4719-927f-d7e4b90fe844)